### ISSUE TITLE
quantile / median method 

### ISSUE THREAD

**Issue by fabian-s** [fabian-s/tidyfun#19]
... not feasible for `feval_irreg` 

... for `feval_reg`:  use MBD? (see `roahd:::MBD.default`, e.g.),  other depths as in `rainbow:::fdepth`, `ddalpha::depthf`?

... for `fbase`: could use multivariate depths on coefs directly (but consider scaling of basis functions)

**Comment by fabian-s** [comment]
@jeff-goldsmith : any preferences / any experience which depths work well and should be the default?

**Comment by jeff-goldsmith** [comment]
i'd vote `roahd::MBD.default` of these options, although i haven't used it much myself. 

i guess we'll trust users to compute quantile / median curves for smooth `feval_reg` data?

**Comment by fabian-s** [comment]
Even cheaper and more intuitive alternative: [half region depth](https://www.sciencedirect.com/science/article/pii/S0167947310004123)

**Comment by fabian-s** [comment]
yeah, I know, `roahd` uses the [trick](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.231.6990&rep=rep1&type=pdf) you probably mean.  I did mean cheaper computationally, but come to think about it this might not actually be true....

**Comment by fabian-s** [comment]
see also experiments in attic/dev-quantile.Rmd
